draft-ietf-quic-recovery.md

```diff
@@ -1202,15 +1205,26 @@ Invoked by loss detection from DetectLostPackets when new packets
 are detected lost.
 
 ~~~
+   InPersistentCongestion(oldest_loss_time):
+     return
```

Comment:
This expression is hard to read, I think it's worth breaking it up some.
 elapsed_time = Now() - oldest_loss_time
 pto = smoothed_rtt + 4 * rttvar + max_ack_delay
 return elapsed_time > pto * (2 ^ kPersistentCongestionThreshold - 1)